controller/injector/k8s_binding_injector_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package injector
+
+import (
+	"testing"
+)
+
+func TestInject(t *testing.T) {
```

Comment:
I made an issue for it (Issue #166), as I really wanted to get this in to get rid of the service-to-service bindings ASAP. If you guys feel it's necessary as part of the PR that's fine, but will take me a bit of time to ramp up on proper way to mock/test against k8s client.